students/15th/byunggonmoon/main.html

```diff
@@ -0,0 +1,124 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>  westagram </title>
+        <link rel="stylesheet" type="text/css" href="main.css">
+</head> 
+
+<body>
+<section class= "mainPage">
+    <div class= "mainBar">
+      <div class= "barOutline">
+        <div class= "logoOutline"> 
+          <div class= "logo"> 
+          </div>
+        </div>
+        <div class= "searchBarOutline">
+           <input class= "searchButton">
+              <span class="searchIcon"></span>
+              <span class="searchWord">검색</span>
+          
+        </div>
+        <div class=iconsOutline>
+         <div class= "icons"> 
+          <div class= "homeIcon"> </div>
+           <div class= "messageIcon"> </div>
+           <div class= "exploreIcon"> </div>
+           <div class= "activityIcon"> </div>
+           <div class= "profileIcon"></div> 
+          </div>
+        </div>
+      </div> 
+    </div>
+
+  <div class= "outLine">
+       <div class= "mainContents">
+          <div class= "stories">
+            <div class= "storyBar">
+              <li>
+              <div class= "storyContents">
+
+              </div> 
+              </li>
```

Comment:
- 들여쓰기 띄어쓰기 지켜서 아래와 같이 작성해주세요! 
```suggestion
              <li>
                <div class= "storyContents"></div> 
              </li>
```
- li태그를 사용하신 이유가 무엇일까요 ? 🤔